Tasks/PublishCodeCoverageResults/publishcodecoverageresults.ts

```diff
@@ -40,12 +40,13 @@ async function run() {
                         additionalFiles,
                         { followSymbolicLinks: false, followSpecifiedSymbolicLink: false },
                         { matchBase: true });
-                    tl.debug(tl.loc('FoundNMatchesForPattern', additionalFileMatches.length, additionalFiles));
                 }
                 else {
                     // Use the specific additional file (no wildcards)
                     var additionalFileMatches: string[] = [additionalFiles];
                 }
+                additionalFileMatches = additionalFileMatches.filter(file => pathExistsAsFile(file));
+                tl.debug(tl.loc('FoundNMatchesForPattern', additionalFileMatches.length, additionalFiles));
             }
```

Comment:
Spell check

Reply:
What happens when no pattern match?

Reply:
additionalcodecoveragefiles value is passed null to command. This is also shown in logs.